KsApi/MockService.swift

```diff
@@ -98,6 +98,9 @@ internal struct MockService: ServiceType {
   fileprivate let fetchProjectActivitiesResponse: [Activity]?
   fileprivate let fetchProjectActivitiesError: ErrorEnvelope?
 
+  fileprivate let liveAuthTokenResponse: LiveAuthTokenEnvelope?
+  fileprivate let liveAuthTokenError: ErrorEnvelope?
```

Comment:
a pattern we started in the live stream api, and something I've done exactly 4 times in `MockService`, is to use the `Result` type instead of two separate variables. wanna give that a shot?